File: 0476-number-complement/0476-number-complement.cpp
class Solution {
public:
    int findComplement(int num) {
        string a=bitset<32>(num).to_string().substr(bitset<32>(num).to_string().find('1'));
        string b;
        for (char i:a){
            if (i=='1'){
                b+='0';
            }else{
                b+='1';
            }
        }
        int x = stoi(b,NULL,2);
        return x;




    }
};

File: 0476-number-complement/0476-number-complement.py
class Solution:
    def findComplement(self, num: int) -> int:
        a = bin(num)[2:]
        b = ''
        for i in a:
            if i=='1':
                b+='0'
            else:
                b+='1'
        x = int(b,2)
        return x